flutter_vlc_player/example/lib/vlc_player_with_controls.dart

```diff
@@ -377,6 +356,28 @@ class VlcPlayerWithControlsState extends State<VlcPlayerWithControls>
     );
   }
 
+  void _cyclePlaybackSpeed() async {
+    playbackSpeedIndex++;
+    if (playbackSpeedIndex >= playbackSpeeds.length) {
+      playbackSpeedIndex = 0;
+    }
+    return await _controller
+        .setPlaybackSpeed(playbackSpeeds.elementAt(playbackSpeedIndex));
+  }
+
+  void _setSoundVolume(value) {
+    setState(() {
+      volumeValue = value;
+    });
+    _controller.setVolume(volumeValue.toInt());
+  }
+
+  void _togglePlaying() async {
+    return _controller.value.isPlaying
+        ? await _controller.pause()
+        : await _controller.play();
```

Comment:
```suggestion
   _controller.value.isPlaying
        ? await _controller.pause()
        : await _controller.play();
```
or even 
```suggestion
    if (_controller.value.isPlaying) {
      await _controller.pause();
    } else {
      await _controller.play();
    }
```
What do you think?

Reply:
I think the ternary should be OK for this one - we're not going to do any extra work, aside from toggling the player. Also, it's a bit more concise.